digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-core/src/main/java/de/muenchen/oss/digiwf/s3/integration/adapter/in/streaming/MessageProcessor.java

```diff
@@ -51,4 +53,28 @@ public Consumer<Message<CreatePresignedUrlEvent>> createPresignedUrl() {
             }
         };
     }
+
+    public Consumer<Message<FilesDTO>> deleteFiles() {
```

Comment:
Wir haben die dazugehörigen element templates sonst unter doc gespeichert, das fehlt hier noch.
Ist es sinnvoll die emlement templates weiterhin dort zu speichern?

Reply:
Vorerst werden Element-Templates doppelt gepflegt. Ziel ist es diese aber als extra Paket in digiwf-core umzusetzen und dieses im BPM-Modeler zu nutzen